docs/03-for-team-members/02-becoming-a-team-member.md

```diff
@@ -0,0 +1,48 @@
+# Team members
+
+Flathub, as a centralized repository of software apps, has a dedicated team of
+contributors necessary to keep it running. These contributors are in charge of:
+
+* Reviewing new applications
+* Reviewing permission changes for existing aplications
+* Requesting permission changes for existent apps that might not fit the
+  guidelines
+
+In consequence, they can be seen as the auditors and moderators of our
+community. As such, they play a big role in the decision-making process.
+They are also an important part of the Flathub and Flatpak trust model. The fact
+that applications' permissions are audited by the community is a helps to ensure
+consistent and safe permissions in the majority of the cases. Therefore, they
+also enjoy the recognition for their work!
+
+## Becoming a team member
+
+Since auditing apps and permissions is a great task, we are always in need of
```

Comment:
I'd just say "we are always in need of more hands" or "Since auditing apps and is a great task, we are always in need of...". 

Realistically it's not going to be _only_ auditing.

Reply:
Thanks! I've got a terribly verbose way of writing..